src/components/formControls/input/style.ts

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+import { Props } from './index'
+import { setTheme } from '../../../helpers'
+
+export const StyledWrapper = styled.div`
+  max-width: ${(p: Props) => setTheme(p.theme, 'maxWidth') || '254px'};
```

Comment:
Does it need a wrapper? This Input component is mostly all 👍 from me since it's not reading css values as inline props, except here. The component can be 100% width, and if the caller wants to restrict its width, it can use its own wrapper element with a fixed width, or inside a flex / grid / whatever layout.

Or, if we want a consistent width for most form components / Input components, then have the '254px' controlled by a slightly more semantic prop, such as `fullWidth: bool`. Then it either gets the easy default, or can be put in a more flex layout by setting that prop to `true`.

Reply:
I would suggest that we create something like FormWrapper, which should be used in the combination with form controls. This way we can have different wrapper types (full width, limited, etc) and controlled in one place for all controls (input, select, checkbox, radio, etc).

Something like https://github.com/brave/brave-ui/blob/62a127bd0f7bea47d547a01717009b3ecf601bf3/src/components/formControls/textarea/index.tsx#L30. But if we would use it we would need to improve ControlWrapper as you suggested, so that we are again not passing px in but type

Reply:
this component is mostly a shallow copy of the current TextArea component with small variations.

we need consensus in this part I'm planning something here. Meanwhile, I'll update this per design spec so we can discuss what/what not a component should be. I'll ping you both

Reply:
Yeah I think that we should pull out `ControlWrapper` from inputs/textarea and developer should wrap it in parent component.